openapi/components/schemas/ThreeDSecureServers/ThreeDSecureIO3dsServer.yaml

```
@@ -1,123 +1,128 @@
-          '02': Instalment transaction.
-          '03': Add card.
-          '04': Maintain card information.
-          '05': Account verification.
-          '06': Split/delayed shipment.
-          '07': Top-up.
-          '08': Mail order.
-          '09': Telephone order.
-          '10': Whitelist status check.
-          '11': Other payment.
+type: object
+required:
+  - name
+  - acquirerMerchantIdVisa
+  - acquirerMerchantIdMastercard
+  - merchantName
+  - merchantAcquirerBinVisa
+  - merchantAcquirerBinMastercard
+  - merchantCountry
+  - merchantUrl
+  - transactionType
+properties:
+  name:
+    type: string
+    description: Merchant plug-in name.
+    enum:
+      - ThreeDSecureIO3dsServer
+  acquirerMerchantIdVisa:
+    type: string
+    description: Visa acquirer merchant ID (MID).
+    maxLength: 35
+  acquirerMerchantIdMastercard:
+    type: string
+    description: Mastercard acquirer merchant ID (MID).
+    maxLength: 35
+  acquirerMerchantIdAmex:
+    type: string
+    description: American Express acquirer merchant ID (MID).
+    maxLength: 35
+  acquirerMerchantIdDiscover:
+    type: string
+    description: Discover/Diners acquirer merchant ID (MID).
+    maxLength: 35
+  acquirerMerchantIdJcb:
+    type: string
+    description: JCB acquirer merchant ID (MID).
+    maxLength: 35
+  merchantName:
+    type: string
+    description: Merchant name.
+    maxLength: 40
+  merchantAcquirerBinVisa:
+    type: string
+    description: Visa acquirer BIN.
+    minLength: 6
+    maxLength: 11
+  merchantAcquirerBinMastercard:
+    type: string
+    description: Mastercard acquirer BIN.
+    minLength: 6
+    maxLength: 11
+  merchantAcquirerBinAmex:
+    type: string
+    description: American Express acquirer BIN.
+    minLength: 6
+    maxLength: 11
+  merchantAcquirerBinDiscover:
+    type: string
+    description: Discover/Diners acquirer BIN.
+    minLength: 6
+    maxLength: 11
+  merchantAcquirerBinJcb:
+    type: string
+    description: JCB acquirer BIN.
+    minLength: 6
+    maxLength: 11
+  merchantCountry:
+    type: string
+    description: Merchant country ISO Alpha-2 code format.
+    maxLength: 2
+    example: US
+  merchantUrl:
+    type: string
+    description: Merchant URL.
+    maxLength: 2048
+  transactionType:
+    type: string
+    enum:
+      - '01'
+      - '03'
+      - '10'
+      - '11'
+      - '28'
+    description: Identifies the type of transaction that is being authenticated.
+    x-enumDescriptions:
+      01: Goods/Service purchase.
+      03: Check acceptance.
+      10: Account funding.
+      11: Quasi-Cash transaction. Indicates the type of 3RI request.
+      28: Prepaid activation and load.
+  declineNotEnrolled:
+    type: boolean
+    description: Specifies whether to decline transactions if a payment card is not enrolled.
+    default: false
+  use3dsForMerchantInitiated:
+    type: boolean
+    description: Use 3DS for merchant initiated transactions.
```

nitpick: I think this description is missing the `specifies...` part
```suggestion
    description: Specifies whether to use 3DS for merchant initiated transactions.
```